release/ray_release/test.py

```diff
@@ -251,15 +251,15 @@ def get_byod_image_tag(self) -> str:
         }
         return f"{self.get_byod_base_image_tag()}-{dict_hash(custom_info)}"
 
-    def _use_byod_ml_image(self) -> bool:
+    def use_byod_ml_image(self) -> bool:
```

Comment:
maybe if using ml image should have it's own field? like technically speaking, we also have `ray-ml:cpu` image, just never used in release tests..